qa/tasks/scripts/clobber_repositories.sh

```diff
@@ -18,15 +18,29 @@ cp -a /etc/zypp/repos.d /etc/zypp/repos.d.bck
 rm -f /etc/zypp/repos.d/*
 
 for repo_spec in "$@" ; do
-    repo_name=$(echo $repo_spec | sed -e 's/\:.*$//')
-    repo_url=$(echo $repo_spec | sed -e 's/^.*\:\(http:.*$\)/\1/')
-    zypper \
-        --non-interactive \
-        addrepo \
-        --refresh \
-        --no-gpgcheck \
-        $repo_url \
-        $repo_name
+    if [[ "$repo_spec" =~ '!' ]] ; then
```

Comment:
Ups, I missed this code, can we provide several improvements for this? I would use bash templates for this instead of sed:

```
  repo_url=${repo_spec#*:}
  repo_name=${repo_spec%%:*}
  if [[ "$repo_name" =~ '!' ]] ; then
      repo_prio=${repo_name#*\!}
      repo_name=${repo_name%\!*}
      zypper . . .
  else
      zypper . . .
  fi
```

Reply:
Also, the head of this file can be updated: remove `TODO: priority`